pimd/pim_nb_config.c

```diff
@@ -2927,18 +2935,19 @@ int lib_interface_gmp_address_family_static_group_create(
 		ifp = nb_running_get_entry(args->dnode, NULL, true);
 		yang_dnode_get_ip(&source_addr, args->dnode, "./source-addr");
 		yang_dnode_get_ip(&group_addr, args->dnode, "./group-addr");
-
-		result = pim_if_igmp_join_add(ifp, group_addr.ip._v4_addr,
-				source_addr.ip._v4_addr);
+#if PIM_IPV == 4
+		result = pim_if_gm_join_add(ifp, group_addr.ip._v4_addr,
```

Comment:
Since this is `struct ipaddr`, it might be better to handle this v4/v6 inside `pim_if_gm_join_add()`?